jsonnet/kube-thanos/kube-thanos-store.libsonnet

```diff
@@ -135,11 +135,16 @@ function(params) {
           { name: 'tls-secret', mountPath: ts.config.objectStorageConfig.tlsSecretMountPath },
         ] else []
       ),
-      livenessProbe: { failureThreshold: 8, periodSeconds: 30, httpGet: {
-        scheme: 'HTTP',
-        port: ts.config.ports.http,
-        path: '/-/healthy',
-      } },
+      livenessProbe: {
+        failureThreshold: 8,
+        periodSeconds: 30,
+        timeoutSeconds: ts.config.livenessProbe.timeoutSeconds,
+        httpGet: {
+          scheme: 'HTTP',
+          port: ts.config.ports.http,
+          path: '/-/healthy',
+        },
```

Comment:
I was torn between making only exactly what I need configurable vs making the whole thing configurable. As a poorly configured liveness probe can cause substantial problems I refrained from making it completely configurable. 

I could make the `failureThreshold` and `periodSeconds` configurable in addition to the `timeoutSeconds`. What do you think, @yeya24?

Reply:
I feel it is fine to expose the field completely. Since we leave the defaults here so users can take their own risk to change to what they need.
But exposing `failureThreshold`, `periodSeconds` and `timeoutSeconds` is also a way to go. Let's expose these fields for now.

Reply:
Done it in https://github.com/thanos-io/kube-thanos/pull/286/commits/449e423dc618f2911038a1b0111ab5a6a7c7174e. 👍